server/services/caseloadService.ts

```diff
@@ -133,27 +130,28 @@ export default class CaseloadService {
       .then(caseload => this.mapResponsibleComsToCases(caseload))
   }
 
-  public pairNomisRecordsWithDelius = async (prisoners: Container<Prisoner>): Promise<Container<ManagedCase>> => {
+  public pairNomisRecordsWithDelius = async (prisoners: Container<CaseloadItem>): Promise<Container<ManagedCase>> => {
     const caseloadNomisIds = prisoners
       .unwrap()
-      .filter(offender => offender.prisonerNumber)
-      .map(offender => offender.prisonerNumber)
+      .filter(({ prisoner }) => prisoner.prisonerNumber)
+      .map(({ prisoner }) => prisoner.prisonerNumber)
 
     const deliusRecords = await this.communityService.getOffendersByNomsNumbers(caseloadNomisIds)
 
     return prisoners
-      .map(offender => {
+      .map(({ prisoner: offender, cvl: cvlFields }) => {
         const deliusRecord = deliusRecords.find(d => d.otherIds.nomsNumber === offender.prisonerNumber)
         if (deliusRecord) {
           return {
             nomisRecord: offender,
+            cvlFields,
```

Comment:
worth keeping in mind that these are always pulled from prisoner offender search rather than the cached version on the licence - will have to work out if that matters/ is ideal. 
Might be we slowly move over to use licences when we start building caseloads in the backend